### DISCUSSION TITLE
lightgallery and Admonition

### DISCUSSION THREAD

**Discussion by foxdie987** [HEIGE-PCloud/DoIt#362] [Q&A] [answered]
I have some pictures that I want to hide by default, to avoid spoiler or something horrific, and should warn viewers first.
It should be hidden until the user click on Admonitions to reveal them.

`{{< image src="/images/pic.png" caption=" " linked=true >}} ` => can be supported by Lightgallery.js, but cannot be used in Admonition, so I have no choice but to use the basic  `![pic.jpg](./images/pic.jpg)` which doesn't seem to work with Lightgallery.js (user can't click on the picture to go to gallery mode)

[Example of folded pictures](https://foxdie.netlify.app/arai2/)

So the question is, can we somehow make **lightgalery** and picture in **Admonition** work together?

**Comment by HEIGE-PCloud** [comment]
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/52968553/143229860-25266a03-8e64-4a6f-813b-0a37d3cdaf25.png">

<img width="1128" alt="image" src="https://user-images.githubusercontent.com/52968553/143229909-c5f2144a-fde9-4606-8a87-44923e63e433.png">

Cannot reproduce, the lightgallary should work just fine with admonition.

<img width="808" alt="image" src="https://user-images.githubusercontent.com/52968553/143230104-8f31cfc5-455b-4548-99ff-512a9fe91135.png">

I think your problem here is that the link to your image is not correct. Maybe double check the file path for your images...

**Reply by foxdie987** [reply]
I forgot to reply here :grinning: please check my answer below.

**Comment by foxdie987** [comment, marked as answer]
Thank you, I check file path but it's correct and images are showing just fine.

I figured it out, I went back and check the document at [https://hugodoit.pages.dev/basic-markdown-syntax/](https://hugodoit.pages.dev/basic-markdown-syntax/), and realized that:
`![Alt text](https://octodex.github.com/images/stormtroopocat.jpg "The Stormtroopocat") ` => recognized by LightGallery by adding a caption.
`![Minion](https://octodex.github.com/images/minion.png)` => it won't be parsed by LightGallery without caption I think.

As for now, the issue went away if I add a caption to images. 

```
{{< admonition type=abstract title="Click to fold" open=false >}}
![alt text](./images/path to picture  "caption text")
{{< /admonition >}}
```

So lightgalery and picture in Admonition works okay though.
